data/org.darktable.darktable.appdata.xml.in

```diff
@@ -11,9 +11,8 @@
   <translation type="gettext">darktable</translation>
   <description>
     <_p>
-      darktable manages your digital negatives in a database and lets you
-      view them through a lighttable. It also enables you to develop raw
-      images and enhance them in a darkroom.
+      darktable manages your images (raw and other formats) in a database, lets you
```

Comment:
I'm not sure which is more accurate. I'd take @elstoc's advice.

Reply:
Strong preference for "image" here (verging on insistence). @TurboGit's argument that a RAW file is not an image also applies to the word "picture" IMO - a RAW file is just "data" so is neither "image" nor "picture". I have used the word "image" throughout the user manual and always try to change "picture" to "image" when I copy-edit both user manual submissions and the release notes.

I would call a "jpeg" or a "png" an "image file" (not sure I've ever heard anyone say "picture file" or "jpeg picture") regardless of what it's an image _of_. An image of a house is definitely a "picture" but, if it's just file of all black pixels it's still an image but is it any longer a "picture"? I expect "picture" to be an actual representation of something, but "image" could be anything as long as it's viewable on a screen.

Darktable processes raw files (or image files) into image files which may or may not be pictures.

Reply:
I am not going to touch on the nuances and differences in their use in English, but I would say that `image` and `picture` are often used interchangeably.   Therefore, if you look at it in a way that a raw file is not an image in a certain sense (not _yet_ an image, you mean?), then it is also not a picture.

Reply:
Agreed image and picture are often used interchangeably (whether they should be is another question), but in the context of the things darktable works with (files) image is more generally correct and accepted terminology.

Reply:
> Agreed image and picture are often used interchangeably (whether they should be is another question), but in the context of the things darktable works with (files) image is more generally correct and accepted terminology.

BTW, I am preparing a PR to replace the use of `picture` (I found a few) in darktable with `image`.

Reply:
> In terms of this text I would say "manages your raw files and images" (maybe "camera raw files"?)

So are you both looking at it like raw files are not images? Because the raw file needs demosaicing? But, on the other hand, images in other formats also require some form of decoding... What's the difference?

Reply:
I guess I would say that most other image formats have been processed in such a way that their visual presentation is intended (by someone) and encoded into the file. Whereas any visual presentation of a raw file is just a representation of data and almost certainly doesn't match a visual intention.

Splitting hairs a bit I know. I've always thought of raw files as just "data captured by the camera that has not been processed for display". Seems kind of more correct and at least once I decided on that distinction it made the user manual more consistent.

Reply:
So, I changed as advised: `darktable manages your camera raw files and images in a database`

Reply:
Raw (undemosaiced) files can definitely be viewed as is, and you will be able to recognise things this way. So I would say it’s an image and may even be a picture;) They are just not intended to be used like that.

Furthermore, most (non-raw) image file formats use encoding or/and compression of some sort, so data first has to be processed before it’s being ready for display. Yes, rules on how to do it are well-defined (and necessary parameters are included in file metadata), but it’s exactly the same for DNG file - there is a specification on how to process them, and metadata fully describes things like colorspace.

So I say the distinction between raw and non-raw lies somewhere else, not in “image” and “not an image”.

Reply:
Yes it's hard to decide where to draw the line linguistically. But I think we can agree that "jpeg image", "png image", "tiff image", "raw file" are reasonable and uncontroversial terms to use. Yes we _could_ use the term "raw image" depending on where we draw that line but given that even we cannot agree where that line should be drawn, I think those terms are consistent and acceptable.